google/cloud/texttospeech/v1/internal/text_to_speech_connection_impl.cc

```diff
@@ -89,6 +89,15 @@ TextToSpeechConnectionImpl::SynthesizeSpeech(
       *current, request, __func__);
 }
 
+std::unique_ptr<::google::cloud::AsyncStreamingReadWriteRpc<
```

Comment:
I am patting myself on the back for #14250

Previously we would have had to done manual work to make a new bidirectional streaming RPC build.